app/components/layouts/components/suggestionList.tsx

```diff
@@ -79,29 +81,43 @@ const SuggestionList: React.SFC<SuggestionListProps> = props => {
         e.preventDefault();
         props.handleClickSuggestionKeyword(props.suggestionList[index]);
       }}
-      className={styles.keywordCompletionItem}
+      className={classNames({
+        [`${styles.keywordCompletionItem}`]: index != 0,
+        [`${styles.highLightKeywordCompletionItem}`]: index === 0,
+      })}
       onKeyDown={e => {
         handleArrowKeyInput(e, props.handleClickSuggestionKeyword, props.suggestionList[index]);
       }}
       key={`keyword_completion_${suggestion}${index}`}
       tabIndex={-1}
     >
-      <span dangerouslySetInnerHTML={{ __html: suggestion }} />
+      <span className={styles.keywordCompletionItemContext} dangerouslySetInnerHTML={{ __html: suggestion }} />
     </a>
   ));
-
+  highlightedList.length;
   return (
     <div style={{ display: props.isOpen ? "block" : "none" }} className={styles.keywordCompletionWrapper}>
       {highlightedContent}
+      {props.userInput.length > 0 ? (
```

Comment:
In this case, I personally prefer to use `props.userInput.length > 0 &&` syntax.